### PR TITLE
Mention early access features in readme

### PR DESCRIPTION
**Pull request by t1m0thyj**
To be merged after #930

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -15,6 +16,15 @@ Zowe CLI is a command-line interface that lets you interact with the mainframe i
...
  - [FAQs](#frequently-asked-questions)
 
+## Early Access Features
+
+To try out early access features, install the "next" release of Zowe CLI (`npm install -g @zowe/cli@next`). This version may receive breaking changes and is intended to gather early feedback on what may become a future LTS release.
+
+For documentation about these features, see these files:
+
+- [Using Global Profile Configuration](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md)
+- [Using Daemon Mode with "zowex"](https://github.com/zowe/zowe-cli/blob/next/zowex/design.md)
```
Comment (gejohnston) [inline packages/cli/README.md:26]: The daemon mode link resulted in 404 "page not found".

Comment (t1m0thyj) [reply packages/cli/README.md:26]: Currently it's in the "json-config-next" branch at https://github.com/zowe/zowe-cli/blob/json-config-next/zowex/design.md. The link should work for the next branch after #930 is merged.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ This repository also contains the Zowe Node Client SDK. The SDK lets you leverag
 
 ## Contents  <!-- omit in toc -->
 
+ - [Early Access Features](#early-access-features)
```
Comment (gejohnston) [inline README.md:11]: This link goes to the right document, but appears to go to the  "Build from source" link.

Comment (gejohnston) [reply README.md:11]: I think a couple of the following links also go to the wrong topic.

Comment (t1m0thyj) [reply README.md:11]: The link works fine for me. Maybe it behaves differently than expected when clicked in the PR diff view?
github-readme-toc

File: packages/cli/README.md (hunk 1 above)
Comment (gejohnston) [reply packages/cli/README.md:26]: Ok. Thanks.

File: README.md (hunk 2 above)
Comment (gejohnston) [reply README.md:11]: Then I am not worried about it.

**Review by gejohnston — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**